src/components/careers/desktop/portfolio_section/ProfileSummary.tsx

```diff
@@ -5,25 +5,43 @@ import {Member} from "../../../../constants/members";
 import {ChipSmall} from "../ChipSmall";
 import {PortfolioLinkBox} from "./PortfolioLinkBox";
 
+const BACKGROUND_PATTERN = process.env.PUBLIC_URL + "/imgs/life-bg-pattern.svg";
+
 const StyledGrid = styled(Grid)({
   width: '18.75vw',
   height: '25vw',
   padding: '2.1vw 1.6vw 1vw 1.6vw',
-  backgroundColor: '#000e3c',
+  backgroundColor: "#191a1a",
   boxSizing: 'border-box',
   userSelect: 'none',
   '-moz-user-select': '-moz-none',
   '-khtml-user-select': 'none',
   '-webkit-user-select': 'none',
   '-ms-user-select': 'none',
   position: 'relative',
-  cursor: "url(imgs/cursors/grab-cursor-icon-small.png), default"
+  cursor: "url(imgs/cursors/grab-cursor-icon-small.png), default",
+  zIndex: 1,
+  '&::after': {
+    content: `""`,
+    position: 'absolute',
+    inset: 0,
+    backgroundImage: `url(${BACKGROUND_PATTERN})`,
+    backgroundPosition: `${Math.random()*100}% ${Math.random()*100}%`,
```

Comment:
@krisbitney I think I need to pass these values in via props so that they calculate at the beginning of each component's lifecycle?